inference-engine/src/mkldnn_plugin/nodes/mkldnn_gather_node.cpp

```diff
@@ -2,30 +2,22 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include <cmath>
 #include <vector>
 #include <string>
 #include <mkldnn_types.h>
 #include "ie_parallel.hpp"
 #include "mkldnn_gather_node.h"
 #include <ngraph/opsets/opset1.hpp>
-#include <precision_utils.h>
 #include "common/cpu_memcpy.h"
 
 using namespace MKLDNNPlugin;
 using namespace InferenceEngine;
 
 bool MKLDNNGatherNode::isSupportedOperation(const std::shared_ptr<ngraph::Node>& op, std::string& errorMessage) noexcept {
     try {
-        auto gatherOp = ngraph::as_type_ptr<const ngraph::op::v1::Gather>(op);
+        auto gatherOp = ngraph::as_type_ptr<const ngraph::op::v7::Gather>(op);
         if (!gatherOp) {
-            errorMessage = "Only opset1 Gather operation is supported";
-            return false;
-        }
-
-        auto axesOp = gatherOp->get_input_node_shared_ptr(GATHER_AXIS);
```

Comment:
Why have you removed this code? Do we support dynamic axes now?

Reply:
In ngraph there is method `get_axis` which has the [check](https://github.com/a-sidorova/openvino/blob/3f8e333baa3adba732925ce2399a2b1992eb9247/ngraph/core/src/op/util/gather_base.cpp#L153) for const axis. Is not it enough and should we check for it in plugin anyway?

Reply:
We should since we want (at least I do) isSupportedOperation to return false in case smt is really unsupported by the plugin

Reply:
Okey, returned. I will know. Thanks!